scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+    sigma_mu = np.sqrt(sigma_f**2 + alleliccapseg_seg_pd['sigma.tau']**2) #we propagate errors in the products f * tau and (1 - f) * tau in the usual way
+    alleliccapseg_seg_pd['mu.minor'] = alleliccapseg_seg_pd['f'] * alleliccapseg_seg_pd['tau']
+    alleliccapseg_seg_pd['sigma.minor'] = sigma_mu
+    alleliccapseg_seg_pd['mu.major'] = (1. - alleliccapseg_seg_pd['f']) * alleliccapseg_seg_pd['tau']
+    alleliccapseg_seg_pd['sigma.major'] = sigma_mu
+
+    #For whatever reason, AllelicCapSeg attempts to call CNLOH.  Documentation is spotty, but it seems like it attempts
+    # to distinguish between three states ("0 is flanked on both sides, 1 is one side, 2 is no cn.loh").
+    # Let's just set everything to 2 for now.
+    # Hopefully, ABSOLUTE is robust to this ...
+    alleliccapseg_seg_pd['SegLabelCNLOH'] = 2
+
+    #One important caveat: for segments with less than 10 hets [TODO: Verify], AllelicCapSeg also tries to call whether a segment is "split" or not.
+    #  This script will attempt to call "split" on all segments.
+    # ACS performs a simple hypothesis test on the alternate-allele fractions to see if
+    # a unimodal distribution peaked at 0.5 is supported over a bimodal distribution peaked at f and 1 - f.
+    # If the former is supported, then AllelicCapSeg ignores the MAP estimate of f and simply sets it to be 0.5.
+    # ABSOLUTE may actually be rather sensitive to this.  Again, let's ignore for now, and we can later port this
+    # statistical test if necessary.  I've done it above (ham-fistedly) and also have python code for the stat test
+    # somewhere.
+
+    #Finally, I believe that ABSOLUTE requires the value of the "skew" parameter from the AllelicCapSeg
+    #allele-fraction model.  This parameter is supposed to allow the model to account for reference bias, though correctness
+    # has been called into question.
+    #  We corrected this during the development of AllelicCNV and retain the same corrected model in ModelSegments.
+    # We will try to transform the relevant parameter in the corrected model back to a "skew",
+    # but this operation is ill defined.  Luckily, for WGS, the reference bias is typically negligible.
+    model_segments_reference_bias = model_segments_af_param_pd[
+        model_segments_af_param_pd['PARAMETER_NAME'] == 'MEAN_BIAS']['POSTERIOR_50']
+    alleliccapseg_skew = 2. / (1. + model_segments_reference_bias)
+
+    return alleliccapseg_seg_pd, alleliccapseg_skew
+
+
+#do the conversion
+alleliccapseg_seg_pd, alleliccapseg_skew = convert_model_segments_to_alleliccapseg(model_segments_seg_pd,
+                                                                                   model_segments_af_param_pd)
+
+#write the results
+alleliccapseg_seg_pd.to_csv(alleliccapseg_seg_output_file, sep='\t', index=False, na_rep='NaN')
+np.savetxt(alleliccapseg_skew_output_file, alleliccapseg_skew)
+
+EOF
+    >>>
+
+    runtime {
+        docker: "continuumio/anaconda"
```

Comment:
This repo seems a little more legit, but perhaps we should still use our own GCR image if appropriate?

Reply:
Again, I do not want to put our GCR image into anything that is seen outside the Broad. 

No action.

Reply:
Might be worth just exposing the Docker images as parameters, then?

Reply:
I did, but only on the tasks.  These will run fine with the GATK docker image (I have tested), so the workflow will feed those in.

More versatile if https://github.com/broadinstitute/cromwell/issues/2912 gets addressed.